packages/deployer/terraform/airnode/main.tf

```diff
@@ -5,61 +5,80 @@
+  source_file        = var.handler_file
+  timeout            = 20
+  configuration_file = var.configuration_file
+  secrets_file       = var.secrets_file
 }
 
 module "callApi" {
   source = "./modules/function"
 
-  name                = "callApi"
-  handler             = "handlers/aws/index.callApi"
-  source_file         = var.handler_file
-  timeout             = 30
-  infrastructure_name = var.infrastructure_name
-  stage               = var.stage
-  airnode_id_short    = var.airnode_id_short
-  configuration_file  = var.configuration_file
-  secrets_file        = var.secrets_file
+  name               = "${local.name_prefix}-callApi"
+  handler            = "handlers/aws/index.callApi"
+  source_file        = var.handler_file
+  timeout            = 30
+  configuration_file = var.configuration_file
+  secrets_file       = var.secrets_file
 }
 
 module "processProviderRequests" {
   source = "./modules/function"
 
-  name                = "processProviderRequests"
-  handler             = "handlers/aws/index.processProviderRequests"
-  source_file         = var.handler_file
-  timeout             = 10
-  infrastructure_name = var.infrastructure_name
-  stage               = var.stage
-  airnode_id_short    = var.airnode_id_short
-  configuration_file  = var.configuration_file
-  secrets_file        = var.secrets_file
+  name               = "${local.name_prefix}-processProviderRequests"
+  handler            = "handlers/aws/index.processProviderRequests"
+  source_file        = var.handler_file
+  timeout            = 10
+  configuration_file = var.configuration_file
+  secrets_file       = var.secrets_file
 }
 
 module "startCoordinator" {
   source = "./modules/function"
 
-  name                = "startCoordinator"
-  handler             = "handlers/aws/index.startCoordinator"
-  source_file         = var.handler_file
-  timeout             = 60
-  infrastructure_name = var.infrastructure_name
-  stage               = var.stage
-  airnode_id_short    = var.airnode_id_short
-  configuration_file  = var.configuration_file
-  secrets_file        = var.secrets_file
+  name               = "${local.name_prefix}-startCoordinator"
+  handler            = "handlers/aws/index.startCoordinator"
+  source_file        = var.handler_file
+  timeout            = 60
+  configuration_file = var.configuration_file
+  secrets_file       = var.secrets_file
 
   invoke_targets                 = [module.initializeProvider.lambda_arn, module.callApi.lambda_arn, module.processProviderRequests.lambda_arn]
   schedule_interval              = 1
   reserved_concurrent_executions = 1
 
   depends_on = [module.initializeProvider, module.callApi, module.processProviderRequests]
 }
+
+module "testApi" {
+  source = "./modules/function"
+
+  name               = "${local.name_prefix}-testApi"
+  handler            = "handlers/aws/index.testApi"
+  source_file        = var.handler_file
+  timeout            = 60
```

Comment:
Probably worth keeping this timeout the same as the `callApi` function's timeout? i.e. 30 seconds. Otherwise, very slow requests might fail to be processed in the node, but testing the endpoint works